tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -531,22 +529,13 @@ def test_check_src_path_pass(self):
         parameters = {}
         for filename in files:
             parameters['dir_op'] = filename[1]
-            cmd_parameter = CommandParameters(self.session, 'put',
-                                              parameters, '')
+            cmd_parameter = CommandParameters('put', parameters, '')
             cmd_parameter.add_region(mock.Mock())
             cmd_parameter.check_src_path(filename[0])
 
-    def test_check_force(self):
```

Comment:
It could be nice if this test was moved to the CommandArchitecture test suite and have something asserted.